### DISCUSSION TITLE
판매한 가격 평균 만들 때

### DISCUSSION THREAD

**Discussion by Seobbi** [Jung110/kaggleteam#14] [Ideas]
1장. 머신러닝 시작하기 - 1절. 머신러닝 개요 에서
robust_scale(x) : 중위수(median)와 사분위범위(interquartile range)를 사용하여 표준화
나중에 판매한 가격 평균 만들 때 사용할 수 있을까요?

**Comment by Jung110** [comment]
robust_scale은 이상치의 영향을 최소로 받는 스케일링 이여서 한번 써보는 것도 좋을 것 같아요
